extensions/wikia/Flags/Flags.i18n.php

```diff
@@ -11,9 +11,11 @@
 $messages['en'] = [
 	'flags-description' => 'Portable flags',
 	'flags-edit-form-more-info' => 'More info >',
-	'flags-edit-modal-title' => 'Flags',
-	'flags-edit-modal-done-button-text' => 'Done',
 	'flags-edit-modal-cancel-button-text' => 'Cancel',
+	'flags-edit-modal-close-button-text' => 'Close',
+	'flags-edit-modal-done-button-text' => 'Done',
```

Comment:
We want to use it for other skins as well. Haven't found any reasonable place to store it.